sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/keyvault/KeyVaultEnvironmentPostProcessorTest.java

```diff
@@ -147,6 +147,18 @@ public void testMultipleKeyVaults() {
         final TokenCredential credentials2 = keyVaultEnvironmentPostProcessorHelper.getCredentials("myvault2");
         assertThat(credentials2, IsInstanceOf.instanceOf(ClientSecretCredential.class));
     }
+
+    @Test
+    public void testGetPropertyFromAzureProperties() {
+        testProperties.put(KeyVaultProperties.getPropertyNameFromAzureProperties(CLIENT_ID), "fake-client-id");
+        propertySources.addLast(new MapPropertySource("Test_Properties", testProperties));
+
+        keyVaultEnvironmentPostProcessorHelper = new KeyVaultEnvironmentPostProcessorHelper(environment);
+
+        final TokenCredential credentials = keyVaultEnvironmentPostProcessorHelper.getCredentials();
+
+        assertThat(credentials, IsInstanceOf.instanceOf(ManagedIdentityCredential.class));
```

Comment:
Can we reuse the logic of `SpringCredentialBuilderBase#populateTokenCredential` in `KeyVaultEnvironmentPostProcessorHelper#getCredentials`?